### PR TITLE
Updated build scripts to produce build artifacts to push to github

### PR DESCRIPTION
**Pull request by david-driscoll**
cc @troydai @DustinCampbell

### CODE REVIEW

**Comment by DustinCampbell** [comment]
LGTM, but take with a grain of salt since I only have a slightly-dangerous understanding of the build scripts

File: build.ps1 [hunk 1]
```diff
@@ -50,22 +50,40 @@ function _test([string] $project, [switch] $skipdnxcore50, [switch] $skipdnx451)
...
-function _publish($project) {
+function _publish($project, $runtime) {
   $name = "$project"
   $src = "src\$project"
   $output = "$publish_output\$project"
 
-  & $dotnet publish $src --framework dnxcore50 --output $output\dnxcore50 --configuration $configuration
-  & $dotnet publish $src --framework dnx451 --output $output\dnx451 --configuration $configuration
+  & $dotnet publish $src --framework dnxcore50 --output $output\$runtime\dnxcore50 --configuration $configuration --runtime $runtime
+  & $dotnet publish $src --framework dnx451 --output $output\$runtime\dnx451 --configuration $configuration --runtime $runtime
```
Comment (troydai) [inline build.ps1]: Can we update the `Readme.md` as well?

**Comment by troydai** [comment]
Changes looks good to me. One issue is that we have too many duplication between batch and bash scripts. Logic for package actually only need on one platform since `dotnet` is able to publish for all platforms from one platform.

**Comment by DustinCampbell** [comment]
Man, it *does* look like bash hates you more than the rest of us.

**Comment by DustinCampbell** [comment]
@david-driscoll -- is there anything I can do to help out here? We're really hoping to get a release of the dev branch.

**Comment by troydai** [comment]
just you know, I’m building the build steps into a compact .net core program so as to consolidate the actions across platforms. In the future when we adopt `Cake` the .net core program can be more easily covered into tasks.

**Comment by troydai** [comment]
Here's the prototype. https://github.com/OmniSharp/omnisharp-roslyn/pull/417
I'll expand this to ps1 tomorrow.

**Comment by troydai** [comment]
@david-driscoll can we close this one?

**Comment by david-driscoll** [comment]
Yeah, we can.

**Pull request closed without merging**